### DISCUSSION TITLE
how do I change the email address associated with my License?

### DISCUSSION THREAD

**Discussion by KarlinTexas** [aclap-dev/video-downloadhelper#119] [License & Purchase requests]
### Your question

I purchased my License more that 8 years ago, and when I moved to a new State, I had to change my email address.
I do not see where I can update my email address associate to my License - through the Firefox extension.

### Addon & CoApp version and details

Using Firefox 122.0.1

**Comment by jcfrog** [comment]
Hi,
please send your old and new email addresses at licenses@downloadhelper.net.
